private/src/scripts/editor/blocks/background-media/deprecated.jsx

```diff
@@ -0,0 +1,23 @@
+import classnames from 'classnames';
+
+const v1 = {
+  save({ className, innerBlocks }) {
+    const leftImage = innerBlocks[0]?.attributes?.image?.id;
+    const rightImage = innerBlocks[1]?.attributes?.image?.id;
+
+    const blockClasses = classnames(className, {
+      'has-imageLeft': !!leftImage,
+      'has-imageRight': !!rightImage,
+    });
+
+    return (
+      <div className={blockClasses}>
+        <InnerBlocks.Content />
```

Comment:
`InnerBlocks` hasn't been imported here, which causes the editor to crash

Reply:
Yeah I spotted that earlier, was hoping I'd get around to it before you did as well 🙈 apologises, I'll fix it now

Reply:
resolved

Reply:
Sorry for beating you to the punch!